packages/markdown/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@prairielearn/markdown",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "main": "./dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PrairieLearn/PrairieLearn.git",
+    "directory": "packages/markdown"
+  },
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch --preserveWatchOutput",
+    "test": "c8 mocha src/**/*.test.ts"
+  },
+  "devDependencies": {
```

Comment:
Conventionally, the order is `dependencies`, then `devDependencies`, then `c8`. If this order is copied from another package, lmk and we can try to fix that one up for consistency.

Reply:
This was copied from path-utils, which doesn't have `dependencies`, so the block was added by `yarn add` at the end. Reordered in e462cf637.